src/storage/assets/priceApiSlice.ts

```diff
@@ -47,8 +47,14 @@ export const priceApi = createApi({
     }),
     getHistoricalPrice: builder.query<HistoricalPriceResult[], HistoricalPriceQueryParams>({
       query: ({ currency, days }) => `/coins/alephium/market_chart?vs_currency=${currency.toLowerCase()}&days=${days}`,
-      transformResponse: (response: { prices: number[][] }) =>
-        response.prices.map((p) => ({ date: dayjs(p[0]).format(CHART_DATE_FORMAT), price: p[1] }))
+      transformResponse: (response: { prices: number[][] }) => {
+        const { prices } = response
+        const today = dayjs().format(CHART_DATE_FORMAT)
+
+        return prices
+          .filter(([date]) => dayjs(date).format(CHART_DATE_FORMAT) !== today)
```

Comment:
Let's just `pop()`

Reply:
I think `filter` is a safer implementation that `pop` x2. WDYT?